src/api/impl/ConnectedServer.cpp

```diff
@@ -118,9 +121,9 @@ static void AppendUrlParam(_Inout_ std::string& sParams, _In_ const char* const
     ra::services::Http::UrlEncodeAppend(sParams, sValue);
 }
 
-Login::Response ConnectedServer::Login(const Login::Request& request) noexcept
+Login::Response ConnectedServer::Login(const Login::Request& request)
 {
-    ra::services::Http::Request httpRequest(m_sHost + "/login_app.php");
+    ra::services::Http::Request httpRequest(ra::StringPrintf("%s/login_app.php", m_sHost.c_str()));
```

Comment:
`ra::StringPrintf` implicitly supports std::string via the variadic templating. You don't need to downcast via `c_str()`.

Reply:
Sweet, that's convenient